### DISCUSSION TITLE
Setting port for PostgreSqlContainer

### DISCUSSION THREAD

**Discussion by grug** [testcontainers/testcontainers-node#763] [Q&A] [answered]
Hey there,

Looking through the source for `PostgreSqlContainer` and `StartedPostgreSqlContainer` - I can't see a way of setting the port. It seems it's always going to be 5432. Am I right in thinking this? Messing around with the code locally, it does seem the case. If I am right, would you accept a pull request from me that makes the port configurable?

**Comment by cristianrgreco** [comment, marked as answer]
Hi, the port is set to 5432 in the constructor, and so is overridable. 
Just invoke `withExposedPorts` on the Postgres container.

**Reply by grug** [reply]
Ah yes - sorry about that!

**Reply by grug** [reply]
Thank you for getting back to me so quickly
